src/Drawing.ts

```diff
@@ -604,7 +604,7 @@ const renderShape: (shape: Shape) => (ctx: CanvasRenderingContext2D) => IO.IO<Ca
                   pipe(
                     shape.closed,
                     B.fold(
-                      () => IO.of(ctx),
+                      () => () => ctx,
```

Comment:
`IO.of(ctx)` was ok

Reply:
See https://github.com/gcanti/graphics-ts/pull/7/commits/d3848bcfd2b47782484a1da84fe31d5e2cae11f8